mindsdb/utilities/config.py

```diff
@@ -254,6 +273,10 @@ def prepare_env_config(self) -> None:
             self._env_config["permanent_storage"] = {"location": "absent"}
         # endregion
 
+        telemetry_enabled = get_bool_env_var("MINDSDB_TELEMETRY")
```

Comment:
maybe to use boolean name? ENABLE_MINDSDB_TELEMETRY or MINDSDB_TELEMETRY_ENABLED